validate.go

```diff
@@ -128,6 +134,26 @@ func checkPlatform(platform rspec.Platform) {
 	logrus.Fatalf("Operation system '%s' of the bundle is not supported yet.", platform.OS)
 }
 
+func checkHooks(hooks rspec.Hooks) {
+	for _, prestart := range hooks.Prestart {
+		if _, err := os.Stat(prestart.Path); err != nil {
+			logrus.Fatalf("Cannot find pre-start hook: %v", prestart.Path)
+		}
+	}
```

Comment:
Probably worth pulling this out into a helper function so it can get more complex without needing copy/pastes to the blocks handling the other hook arrays.  `checkEventHookPaths(hooks []Hook)`?  And this should also check that at least one of the three [executable bits is set][1].  I don't see constants for the bits in `ModePerm`, and I'm barely literate in Go, but `fileInfo.Mode() & 0111` is probably close.

And having validation die at the first error is a bit lazy, but fixing that is probably out of scope for this PR ;).

[1]: https://golang.org/pkg/os/#FileMode